docs/sources/configure-client/trace-span-profiles/ruby-span-profiles.md

```diff
@@ -2,7 +2,7 @@
 title: Span profiles with Traces to profiles for Ruby
 menuTitle: Span profiles with Traces to profiles (Ruby)
 description: Learn about and configure Span profiles with Traces to profiles in Grafana for the Ruby language.
-weight: 100
+weight: 102
 ---
 
 # Span profiles with Traces to profiles for Ruby
```

Comment:
The same intruduction should be added here as well for consistency. Also, if the same OTel section applies.